src/index.js

```diff
@@ -41,6 +41,7 @@ export default class UrlPrettifier<T: string> {
       currentRoute.page === pageName)[0];
     return {
       href: `/${pageName}${this.paramsToQueryString(params)}`,
+      ...(!route && {prefetch: false}),
```

Comment:
I'm not confortable with a lib changing default params for other libs, so we could pass it as an option, what do you think?
Something like:
`new UrlPrettifier(routes, {preventPrefetchIfNoRoute: true})` (`preventPrefetchIfNoRoute` being `false` by default)

Reply:
Sound nice! 
I will update an MR.

Reply:
I don't quite like that you need to pass a property with such a long name, presumably each time you are using component … but I can't come with better naming, so I guess it's a good solution.

Reply:
@BDav24 updated PR :)

Reply:
> I don't quite like that you need to pass a property with such a long name, presumably each time you are using component …

Yes my first idea was to pass the param to the lib entrypoint `new UrlPrettifier(routes, {preventPrefetchIfNoRoute: true})`, so you don't have to pass it each time ;)